tasks/spec_runner.rake

```diff
@@ -55,6 +58,8 @@ module RuboCop
     class ParallelRunner < ::TestQueue::Runner
       def initialize(rspec_args)
         super(Framework.new(rspec_args))
+
+        @exit_when_done = true
```

Comment:
Oooops, this is embarrasing! I'm still wondering what happened but I accidentally lost my local changes right before opening this PR, so I wrote them back from memory... :laughing:, and I wrote the incorrect value for the assignment here :flushed:. Follow up PR [here](https://github.com/rubocop-hq/rubocop/pull/6434).